src/components/Connections/CreateConnectionForm.tsx

```
@@ -578,116 +621,193 @@ const CreateConnectionForm = ({
+                                                  onChange={(event) =>
+                                                    updateColumnSelection(
+                                                      stream.name,
+                                                      col.name,
+                                                      event.target.checked
+                                                    )
+                                                  }
+                                                />
+                                              </TableCell>
+                                              <TableCell>{col.name}</TableCell>
+                                              <TableCell align="right">
+                                                {type === 'string' && <> String</>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)